amazon_payments/views.py

```diff
@@ -648,7 +648,7 @@ def post(self, request, *args, **kwargs):
             data['msg'] = _("You need to add some items to your basket to check out.")
         else:
             try:
-                amazon_order_details = self.get_amazon_order_details(request)
+                amazon_order_details = self.get_amazon_order_details(request, validate_payment_details=False)
```

Comment:
with amazon we don't validate anything they send us a reference token we can use to get information back.( their iframes basically send over the information and for us to get  back any data we have to talk to their servers) unless a user keeps changing their addresses which would be a form of attack I do no think that the few checks we do would hurt our reputation since we can't check up on their iframes.